pkg/sink/sink_test.go

```diff
@@ -1140,7 +1141,13 @@ func (f *echoInterceptor) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	defer r.Body.Close()
+	if f.counter == 0 {
+		data["extensions"].(map[string]interface{})["truncated_sha"].(map[string]interface{})["gone"] = "key"
```

Comment:
Can you change the `truncated_sha` key to something more realistic? E.g. `my_interceptor`